server_nginx.html

```diff
@@ -15,36 +15,40 @@
 # Wide-open CORS config for nginx
 #
 location / {
+     # the dance you have to do because if's can't be nested in nginx
+     if ($http_origin) {
+        set $cors "${cors}O"
```

Comment:
Instead of abbreviations such as 'O', 'P', etc, why not spell things out, e.g. OriginOptions, OriginGet, OriginPost. It would make the code more readable below.